scripts/devops_tasks/test_run_samples.py

```diff
@@ -100,6 +100,18 @@
         "mgmt_topic_async.py",
         "proxy_async.py",
         "receive_deferred_message_queue_async.py"
+    ],
+    "azure-communication-chat": [
```

Comment:
How about SMS and identity, are they already runnable?

Reply:
Yes, they are. These changes are just for which samples to ignore

Reply:
Actually you did make change to the SMS samples as well so not sure why that is not included here?